src/serialization.cpp

```diff
@@ -90,7 +91,7 @@ Status serializeBlobToTensorProto(
         }
         responseOutput.mutable_tensor_shape()->add_dim()->set_size(dim);
     }
-    responseOutput.mutable_tensor_content()->assign((char*)blob->buffer(), blob->byteSize());
+    responseOutput.mutable_tensor_content()->assign(InferenceEngine::as<InferenceEngine::MemoryBlob>(blob)->rwmap().as<char*>(), blob->byteSize());
```

Comment:
rmap()?

Reply:
Tests are failing with `rmap` instead of `rwmap`. `wmap` also fails.

Reply:
Which tests exactly?

Reply:
So the problem was that rmap() was mocked in MockBlob class and 1 test was failing due to rmap() not returning value. Instead of mocking this function I did an override. Now it is working corretly with rmap().